chapters/arrays.tex

```diff
@@ -1434,9 +1434,9 @@ \section{Empty Arrays}\label{empty-arrays}
 
 Multiplication of two empty matrices results in a zero matrix of corresponding numeric type if the result matrix has no zero dimension sizes, i.e.,
 \begin{lstlisting}[language=modelica]
-Real[0, m] * Real[m, n] = Real[0, n] (empty matrix)
-Real[m, n] * Real[n, 0] = Real[m, 0] (empty matrix)
-Real[m, 0] * Real[0, n] = fill(0.0, m, n) (non-empty matrix, with zero elements).
+Real[0, m] * Real[m, n] = Real[0, n] // empty matrix
+Real[m, n] * Real[n, 0] = Real[m, 0] // empty matrix
+Real[m, 0] * Real[0, n] = fill(0.0, m, n) // non-empty matrix, with zero elements.
```

Comment:
Consistent use of punctuation and reformulating to avoid ambiguity:
```suggestion
Real[m, 0] * Real[0, n] = fill(0.0, m, n) // non-empty matrix of zeros
```